akka-bench-jmh/src/main/scala/akka/actor/ActorThoughputBenchmark.scala

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (C) 2014-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.actor
+
+import akka.pattern.ask
+import akka.pattern.pipe
+import akka.testkit.TestProbe
+import com.typesafe.config.ConfigFactory
+import org.openjdk.jmh.annotations._
+import scala.concurrent.duration._
+import java.util.concurrent.TimeUnit
+import scala.concurrent.Await
+import scala.annotation.tailrec
+import java.util.concurrent.CountDownLatch
+import akka.actor.SupervisorSpec.PingPongActor
+import akka.util.Timeout
+import scala.concurrent.Future
+import scala.concurrent.forkjoin.ThreadLocalRandom
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Array(Mode.Throughput))
+@Fork(1)
+@Threads(1)
+@Warmup(iterations = 10, time = 5, timeUnit = TimeUnit.SECONDS, batchSize = 1)
+@Measurement(iterations = 10, time = 15, timeUnit = TimeUnit.SECONDS, batchSize = 1)
+class ActorThroughputBenchmark {
```

Comment:
@zaharidichev I saw something somewhat surprising when debugging how many messages were in the mailbox when running the ForkJoinActorBenchmark and noticed that it's often almost empty (many times 1 message). I was thinking that it perhaps was due to some oscillation effects of the pathological ping-pong, so I wrote this new benchmark. Should be somewhat more realistic. I couldn't see any better behavior in the mailbox though. Should be a good additional benchmark too run with the affinity pool.

Review is welcome.

cc @viktorklang